source/includes/api/_interactions.md

```diff
@@ -0,0 +1,532 @@
+# Interactions
+
+## The interaction object
+
+```json
+{
+  "customerId": "e3899af9-2b74-45dd-b2b1-d7be8ad47bae",
+  "data": {
+    "campaignId": "8b385b44-1618-4dce-bffc-8075002e29a0",
+    "contactId": "7fbec30c-b576-42bb-b5ba-646972844253",
+    "distance": 9.97,
+    "listId": "2210d9ea-1870-4b8d-8803-ce426e33a96b",
+    "location": {
+      "confidence": 0,
+      "lat": 42.3659207,
+      "lng": -71.0587026
+    },
+    "successful": false,
+    "survey": {
+      "responses": {
+        "6430c7c8-364c-4fa5-b145-9dbaa2795f7d": {
+          "fields": {"dogliker": "true"},
```

Comment:
lets change these questions, only keep two and change the examples to something more generic.